app/src/main/res/values/strings.xml

```diff
@@ -78,6 +78,7 @@
     </string-array>
 
     <string name="pdf_ext" translatable="false">.pdf</string>
+    <string name="xls_ext" translatable="false">.xls</string>
```

Comment:
why do you need this twice? You have already defined this value in constants file also
excelExtension